CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The most recent changes are on top, in each type of changes category.
 - Add parentName to basic institution response
 - Add Institution form for superadmin [#470](https://github.com/cyfronet-fid/sat4envi/issues/470)
 - Add institution selection into super Admin and Admin settings page [#455](https://github.com/cyfronet-fid/sat4envi/issues/455)
+- Add generic list view, overhoul institution list view [#456](https://github.com/cyfronet-fid/sat4envi/issues/456)
```

Comment:
For me, it's not necessary, but as we have spoken, new issues should be placed at the top of the list.

Reply:
I remembered that there is some ordering, I just couldn't remember whether it's ascending or descending... :P

Reply:
It isn't so obvious XD